index.bs

```diff
@@ -696,11 +922,28 @@ partial interface NeuralNetworkContext {
             with the value -1 is computed so that the total size remains
             constant.
 
-    **Returns:**: an {{Operand}}. The output tensor. The values of the output
+    **Returns:** an {{Operand}}. The output tensor. The values of the output
     tensor are the same as values of the input tensor. The shape of the output
     tensor is specified by the *newShape* argument.
 </div>
 
+### slice ### {#api-neuralnetworkcontext-slice}
+Produce a slice of the input tensor.
+<script type=idl>
+partial interface NeuralNetworkContext {
+  Operand slice(Operand input, sequence<long> starts, sequence<long> ends);
```

Comment:
Adding a sequence<long> axes may be helpful.

Reply:
Added